### ISSUE TITLE
CloudFormation doesn't work with hyphens

### ISSUE THREAD

**Issue by nordri** [OpenVidu/openvidu#131]
check out if you name the CloudFormation stack with a hyphen it doesn't work.

**Comment by nordri** [comment]
Hi @krish-dev 
I can't see any hyphens (-) in you names.
Anyway, that situation is tested in our CI system and it works. So would you mind, please, provide any information? That is probably happening because the copy/paste of the certificate, check on that!
Thank you very much.

**Comment by krish-dev** [comment]
Sorry for the posting on the wrong thread. apologies for that. Let me try again and I will post the log into the right thread. Thanks
